docs/docsite/Makefile.sphinx

```diff
@@ -6,7 +6,11 @@ SPHINXOPTS    = -j $(CPUS) -n -w rst_warnings
 SPHINXBUILD   = sphinx-build
 SPHINXPROJ    = sdfsdf
 SOURCEDIR     = rst
-BUILDDIR      = _build
+
+# Sets the build output directory if it's not specified on the command line
```

Comment:
This needs to be in the main Makefile- clean won't include this, so will bomb that $(BUILDDIR) is not set